t/matchit_spec.vim

```diff
@@ -6,6 +6,8 @@ runtime plugin/conflict_marker.vim
 let s:lines = [
             \ "<<<<<<< HEAD",
             \ "ourselves1",
+            \ "||||||| merged common ancestors",
+            \ "common ancestors1",
```

Comment:
Would you add another test case rather than modifying current one? I want to test both the cases where `||||||||` exists and where it does not exist.

Reply:
Fixed here
388c7f8